error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
>  You change the default of the whole package, for everybody using this package.

If you're passing an instance of the variable instead, you can still mutate its state and the result is similar. This gives false feeling of safety IMO.

> Imagine somebody does this with a dependency you are relying on and everything ends up in your project's vendor directory.

I doubt any sane implementor of a library would do that.

---

I don't want to argue. I see your point, but I just want make operatorkit to be customisable by external users, while keeping our defaults.

---
 
I added this hackery: https://github.com/giantswarm/microerror/pull/3/commits/10ab4bea94c4bbdff658d6d96a8c95c2ae0753db

This makes `Default` usable, outside the `Mask` functions. LMK if you like it.
If you do: I'll add tests for `Default`.
If you don't: I revert that and make `Default` private, because I want to keep it going.